defaults/include.txt

```diff
@@ -1,3 +1,2 @@
 Wuhan/Hu-1/2019
 Wuhan-Hu-1/2019
```

Comment:
I was curious about the difference between `Wuhan/Hu-1/2019` vs. `Wuhan-Hu-1/2019`, and saw it's explained here: 429a1806e395a90e392ea27d588eb7aea017839a

Reply:
Yeah. Really datasets shouldn't ever have both, but it's still annoying to deal with.